BigBaseV2/src/gta/VehicleValues.h

```diff
@@ -386,9 +386,44 @@ enum VehicleModType
 
 enum CustomVehicleModType
 {
-    MOD_WHEEL_TYPE = -101,
-    MOD_PLATE_STYLE = -102,
-    MOD_WINDOW_TINT = -103
+    MOD_MODEL_HASH        = -100,
+
+    MOD_WHEEL_TYPE        = -101,
+    MOD_PLATE_STYLE       = -102,
+    MOD_WINDOW_TINT       = -103,
+
+    MOD_FRONTWHEEL_VAR    = -104,
+    MOD_REARWHEEL_VAR     = -105,
+
+    MOD_NEON_FRONT_ON     = -106,
+    MOD_NEON_BACK_ON      = -107,
+    MOD_NEON_LEFT_ON      = -108,
+    MOD_NEON_RIGHT_ON     = -109,
+    MOD_TIRE_CAN_BURST    = -110,
+
+    MOD_PRIMARY_COL       = -111,
+    MOD_SECONDARY_COL     = -112,
+    MOD_PEARLESCENT_COL   = -113,
+    MOD_WHEEL_COL         = -114,
+    MOD_INTERIOR_COL      = -115,
+    MOD_DASHBOARD_COL     = -116,
+    MOD_XENON_COL         = -117,
+
+    MOD_PRIMARY_COL_R     = -118,
+    MOD_PRIMARY_COL_G     = -119,
+    MOD_PRIMARY_COL_B     = -120,
+
+    MOD_SECONDARY_COL_R   = -121,
+    MOD_SECONDARY_COL_G   = -122,
+    MOD_SECONDARY_COL_B   = -123,
+
+    MOD_TIRESMOKE_COL_R   = -124,
+    MOD_TIRESMOKE_COL_G   = -125,
+    MOD_TIRESMOKE_COL_B   = -126,
+
+    MOD_NEON_COL_R        = -127,
+    MOD_NEON_COL_G        = -128,
+    MOD_NEON_COL_B        = -129,

```

Comment:
Why are you using negative values? Is it just to not have an index collision in your std::map?

I don't have a problem with it, just odd choice...

Couldn't you use an enum class otherwise and not use manually set enum values and let the compiler decide? It just seems unnecessary to do it like this.

Anyways I'd like to just know your opinion on this.

Reply:
> Why are you using negative values? Is it just to not have an index collision in your std::map?
Yes, I pick negative values to avoid collision. The positive side is used by VehicleModType so I picked the negative side.

> Couldn't you use an enum class otherwise and not use manually set enum values and let the compiler decide? It just seems unnecessary to do it like this.
I guess you are right. It is probably unnecessary and just my personal preference. This structure is actually a preparation for the clone/save current vehicle feature I'm going to implement. I usually set values manually if the structure is going to be serialized so it won't mess with the serialization if I need to update it.

Reply:
Alright, you can keep it as-is then.

Reply:
> @aa15032261 Please note that @GibsonHF's fork already has the feature.

Thanks for letting me know! I'll have a look